tests/e2e/utils/prometheus/setup_prometheus_server.py

```diff
@@ -0,0 +1,207 @@
+import subprocess
+import time
+import re
+import os
+import boto3
+import tempfile
+from e2e.utils.prometheus.createPolicy_AMPIngest import *
+from e2e.utils.utils import wait_for
+from e2e.fixtures.cluster import associate_iam_oidc_provider
+
+prometheus_yaml_files_directory = "../../deployments/add-ons/prometheus"
+default_prometheus_query = 'experiment_server_create_requests'
+local_prometheus_port_forwarding_port = "9090"
+
+def replace_params_env_in_line(file_path, original_to_replacement_dict):
+    updated_file_contents = ""
+    with open(file_path, 'r') as original_file:
+        for line in original_file:
+            for key in original_to_replacement_dict:
+                if key == line.split("=")[0]:
+                    line = f'{key}={original_to_replacement_dict.get(key)}\n'
+            updated_file_contents += line
+    with open(file_path, 'w') as updated_file:
+        updated_file.write(updated_file_contents)
+
+def update_params_env(workspace_id, region, params_env_file_path):
+    original_to_replacement_dict = {}
+    original_to_replacement_dict['workspaceId'] = workspace_id
+    original_to_replacement_dict['workspaceRegion'] = region
+    replace_params_env_in_line(params_env_file_path, original_to_replacement_dict)
+
+def create_AMP_workspace(region):
+    amp_client = boto3.client('amp', region_name=region)
+    
+    workspace_id = amp_client.create_workspace(alias = 'test_AMP_workspace')['workspaceId']
+    print(f"Created Workspace ID: {workspace_id}")
+
+    def check_active_workspace_status():
+        workspace_status = amp_client.describe_workspace(workspaceId=workspace_id).get("workspace").get("status").get("statusCode")
+        print(f"Workspace Status: {workspace_status}")
+        assert "ACTIVE" == workspace_status
+    
+    wait_for(check_active_workspace_status)
+    
+    return workspace_id
+
+def patch_notebook_for_prometheus():
```

Comment:
Is it possible to include the notebook patches in the `patchStrategiesMerge` section so they are applied with the kustomize build/apply command?

Ex: https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/apps/pipeline/s3/kustomization.yaml#L20